taxcalc/calculate.py

```diff
@@ -58,6 +58,9 @@ def __init__(self, parameters, records, **kwargs):
         self._records = (records if not isinstance(records, str) else
                          Records.from_file(records, **kwargs))
 
+        while self._records.current_year < self._parameters.current_year:
```

Comment:
Can we have a keyword arg that would specify this behavior? I'm fine if the default is true. Something like:

```
calc = Calculator(parameters=p, records=r, sync_years=True)
```

I'm concerned about unintended consequences here.